docs/references/rest-apis/rest-inventory-api.md

```diff
@@ -0,0 +1,287 @@
+    ]
+}
+```
+
+#### Start bundle
+- Method: POST
+- API PATH: `/services/inventory/v1/bundles/_start`
+##### Request Body
+``` JSON
+{ 
+"name":"org.eclipse.osgi",
+}
+```
+##### Responses
+- 200 OK status
+- 400 Bad Request (Malformed Client JSON)
+- 404 Resource Not Found
+- 500 Internal Server Error
+
+#### Stop bundle
+- Method: POST
+- API PATH: `/services/inventory/v1/bundles/_stop`
+##### Request Body
+``` JSON
+{ 
+"name":"org.eclipse.osgi",
+}
+```
+##### Responses
+- 200 OK status
+- 400 Bad Request (Malformed Client JSON)
+- 404 Resource Not Found
+- 500 Internal Server Error
+
+---
+
+
+
+## Deployment Packages
+#### GET packages
+- Method: GET
+- Description: returns a list of deployment packages.
+- API PATH: `/services/inventory/v1/deploymentPackages`
+##### Response
+- 200 OK
+```JSON
+{
+    "deploymentPackages":[
+        {
+            "name":"org.eclipse.kura.example.beacon",
+            "version":"1.0.500",
+            "signed":false,
+            "bundles":[
+                {
+                    "name":"org.eclipse.kura.example.beacon",
+                    "version":"1.0.500",
+                    "id": 171,
+                    "state": "ACTIVE",
+                    "signed": false
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+## System Packages (DEB/RPM/APK)
+#### GET System Packages
+- Method: GET
+- Description: returns a list of system packages.
+- API PATH: `/services/inventory/v1/systemPackages`
+##### Response
+- 200 OK
+```JSON
+{
+    "systemPackages":[
+        {
+            "name":"adduser",
+            "version":"3.118",
+            "type":"DEB"
+        },
+        {
+            "name":"alsa-utils",
+            "version":"1.1.8-2",
+            "type":"DEB"
+        },
+        {
+            "name":"ansible",
+            "version":"2.7.7+dfsg-1",
+            "type":"DEB"
+        },
+        {
+            "name":"apparmor",
+            "version":"2.13.2-10",
+            "type":"DEB"
+        },
+        {
+            "name":"apt",
+            "version":"1.8.2.1",
+            "type":"DEB"
+        },
+        {
+            "name":"apt-listchanges",
+            "version":"3.19",
+            "type":"DEB"
+        },
+        {
+            "name":"apt-transport-https",
+            "version":"1.8.2.2",
+            "type":"DEB"
+        },
+        {
+            "name":"apt-utils",
+            "version":"1.8.2.1",
+            "type":"DEB"
+        }
+    ]
+}
+```
+
+ --- 
+## Containers
+#### GET Containers
+- Method: Get
+- Description: returns a list of Containers.
+- API PATH: `/services/inventory/v1/containers`
+##### Response
+- 200 OK
+```JSON
+{
+  "containers":
+  [
+    {
+      "name":"container_1",
+      "version":"nginx:latest",
+      "type":"DOCKER",
+      "state":"active"
+    }
+  ]
+}
+```
+
+#### Start container
+- Method: POST
+- API PATH: `/services/inventory/v1/containers/_start`
+##### Request Body
+``` JSON
+{
+    "name":"container_1"
```

Comment:
It looks like the `version` field is mandatory for the container `_start` and `_stop` requests, otherwise the call will fail with 400 Bad Request